src/libsndfile-1.def

```diff
@@ -1,6 +1,4 @@
-; Auto-generated by cmake
-
-LIBRARY ${PROJECT_NAME}-${PROJECT_VERSION_MAJOR}.dll
+LIBRARY libsndfile-1.dll
```

Comment:
It was correct.

I thought that this file is changing very rarely, so there is not much point in configuring it every time.

But of course i can make a single template for an Autotools and CMake and configure it with something like `PACKAGE` and `VERSION_MAJOR`.

I know how to do this with CMake, but can you give me `VERSION_MAJOR` regex? I see `VERSION_MINOR` in `configure.ac`, something like that. I'm not very friendly with regular expressions.